modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java

```diff
@@ -275,6 +293,56 @@ private void _validateObjectFieldSettingMaximumFileSize(
 		}
 	}
 
+	private void _validateObjectFieldSettingShowFiles(
+			String objectFieldName, String showFilesInDocumentsAndMedia,
+			String storageFolder)
+		throws PortalException {
+
+		if (Validator.isNull(showFilesInDocumentsAndMedia)) {
+			throw new ObjectFieldSettingValueException.MissingRequiredValues(
+				objectFieldName,
+				Collections.singleton("showFilesInDocumentsAndMedia"));
+		}
+		else if (StringUtil.equalsIgnoreCase(
+					showFilesInDocumentsAndMedia, StringPool.TRUE)) {
+
+			_validateObjectFieldSettingStorageFolder(
+				true, objectFieldName, storageFolder);
+		}
+		else if (StringUtil.equalsIgnoreCase(
+					showFilesInDocumentsAndMedia, StringPool.FALSE)) {
+
+			_validateObjectFieldSettingStorageFolder(
+				false, objectFieldName, storageFolder);
+		}
```

Comment:
I've tried, but I don't see a way. Do you have any suggestions?

Reply:
The only thing I could think of was:
```
else if (StringUtil.equalsIgnoreCase(
            showFilesInDocumentsAndMedia, StringPool.FALSE) ||
        StringUtil.equalsIgnoreCase(
            showFilesInDocumentsAndMedia, StringPool.TRUE)) {

    _validateObjectFieldSettingStorageFolder(
        GetterUtil.getBoolean(showFilesInDocumentsAndMedia), 
        objectFieldName, storageFolder);
}
```
but this might not work for a Rest API user, right? Let's keep it the way it is.

Reply:
It would work for an API user, because the if clause guarantees that it's either "true" or "false" and GetterUtil's getBoolean is not case sensitive. I'll do your suggestion. Thanks, @carolmariaabb